src/views/sidebar/components/tabs/summary-tab.js

```diff
@@ -105,7 +96,8 @@ export default class SummaryTab extends Component {
                         inputProps={{ placeholder: '' }}
                     />
                 </section>
-                <DescriptionBox {...descriptionProps} />
+                <span className='summary-tab-category'>Description</span>
+                <DescriptionBox {...this.props} />
```

Comment:
all of them. it's not ideal, but the alternative is the large list we had before. not sure which is better honestly.